lib/fastlane/actions/carthage.rb

```diff
@@ -26,16 +25,21 @@ def self.description
 
       def self.available_options
         [
+          FastlaneCore::ConfigItem.new(key: :command,
+                                       env_name: "FL_CARTHAGE_COMMAND",
+                                       description: "Carthage command (one of `build`, `bootstrap`, `update`)",
+                                       default_value: 'bootstrap',
+                                       verify_block: proc do |value|
+                                         raise "Please pass a valid command. Use one of the following: build, bootstrap, update" unless %w(build bootstrap update).include? value
+                                       end),
           FastlaneCore::ConfigItem.new(key: :update,
```

Comment:
what do you think about this? @ohwutup
both of you think `update` option can be removed, then i will remove it.
some of you think it is needed, then i will mark this as deprecated.

it is ok for me as long as this action has `command` option.